lib/bone.js

```diff
@@ -948,7 +947,13 @@ class Bone {
       };
     }
 
-    Object.defineProperties(this.prototype, descriptors);
+    const customDescriptors = {};
+    Object.keys(descriptors).map(key => {
+      const descriptor = Object.getOwnPropertyDescriptor(descriptors, key);
+      customDescriptors[key] = descriptor;
+    });
```

Comment:
You can grab all the descriptors using [Object.getOwnPropertyDescripts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptors) directly.

I think we can call the third argument `overrides`, e.g. `init(attributes, opts, overrides)`, and then `const descriptors = Object.getOwnPropertyDescripts(overrides)` here.

btw, as we discussed, is `configurable: true` still necessary here?